lib/auth/local/configuration_test.go

```diff
@@ -52,56 +53,35 @@ func (s *ClusterConfigurationSuite) TearDownTest(c *check.C) {
 	c.Assert(s.bk.Close(), check.IsNil)
 }
 
-func (s *ClusterConfigurationSuite) TestAuthPreference(c *check.C) {
```

Comment:
These have been moved to [lib/auth/local/local_test.go](https://github.com/gravitational/teleport/blob/dmitri/split-client-server/lib/auth/local/local_test.go#L1007)